src/utils/functions.ts

```diff
@@ -89,6 +94,31 @@ const checkTheLoggedInUser = async (email: string): Promise<string | null> => {
   }
 };
 
+const isAdmin = (email: string): boolean => {
+  return admins.includes(email);
+};
```

Comment:
check admin password here. for all apis that need admin access implement password